clang/lib/Sema/SemaOverload.cpp

```diff
@@ -2192,6 +2194,9 @@ static bool IsStandardConversion(Sema &S, Expr* From, QualType ToType,
              From->isIntegerConstantExpr(S.getASTContext())) {
     SCS.Second = ICK_Compatible_Conversion;
     FromType = ToType;
+  } else if (ToType->isFixedPointType() || FromType->isFixedPointType()) {
+    SCS.Second = ICK_Fixed_Point_Conversion;
```

Comment:
I guess there's not really any value in breaking this down here?  Seems reasonable.

Reply:
Yeah relating to the block in SemaExprCXX seemed easiest to me.